### PR TITLE
Update actionable.md to iOS 13 behavior

### PR DESCRIPTION
**Pull request by joogps**

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @joogps, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;companion.home-assistant#207).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by TomBrien** [comment]
We should really remove references to 3D Touch. iOS 13 pretty much removed it fully. The interactions is now tap and hold and a haptic is given as feed back.

File: docs/notifications/actionable.md [hunk 1]
```diff
@@ -210,6 +210,10 @@ Notes ![iOS](/assets/apple.svg):
...
+*   For devices that support "Force Touch" / "3D Touch" - a firm press on the notification will expand it, showing the action buttons underneath. Supported devices include the iPhone 6S, iPhone 6S Plus, iPhone 7, iPhone 7 Plus, iPhone 8, iPhone 8 Plus, iPhone X, iPhone XS, iPhone XS Max and Apple Watch models.
 
-*   For devices that do not support "Force Touch" (such as the iPhone 6 and below, iPhone SE, iPhone XR and some iPad models), you instead perform a left to right swipe on the notification, then tap on the 'View' button. This will expand the notification and show the relevant action buttons underneath.
+*   For devices that do not support "Force Touch" (such as the iPhone 6 and below, iPhone SE, iPhone XR, iPhone 11 and later, and some iPad models):
+
+       **Prior to iOS 13:** You perform a left to right swipe on the notification, then tap on the 'View' button. This will expand the notification and show the relevant action buttons underneath.
```
Comment (TomBrien) [inline docs/notifications/actionable.md]: This is a lock screen notification and still present in iOS 13 (I believe)

File: docs/notifications/actionable.md [hunk 2]
```diff
@@ -210,6 +210,10 @@ Notes ![iOS](/assets/apple.svg):
...
+  
+       **iOS 13 and later:** You press and hold the notification. This will expand the notification and show the relevant action buttons underneath.
```
Comment (TomBrien) [inline docs/notifications/actionable.md]: This is for banner notifications

**Comment by joogps** [comment]
Maybe it's clearer now? Actually, I think we don't even need this explanation. Home Assistant users certainly know how iOS works, right? Also, I double-checked and, yes, even in iOS 13, 3D Touch is used when expanding notifications.

**Comment by TomBrien** [comment]
Looking good. I just tested on my Xs and the lightest of touch and hold will expand the notification. SO I would favour a single instruction set for all iOSs. 

> Home Assistant users certainly know how iOS works, right?
You would think, but not always. To be honest in the days of FaceID staying on the lock screen is pretty hard and the pull down to open a banner is not intuitive

File: docs/notifications/actionable.md [hunk 3]
```diff
@@ -210,10 +210,12 @@ Notes ![iOS](/assets/apple.svg):
 
 ![iOS](/assets/apple.svg) iOS Specific
 
-*   For devices that support "Force Touch" / "3D Touch" - a firm press on the notification will expand it, showing the action buttons underneath. Supported devices include the iPhone 6S, iPhone 6S Plus, iPhone 7, iPhone 7 Plus, iPhone 8, iPhone 8 Plus, iPhone X, iPhone XS, iPhone XS Max and Apple Watch models.
+### iOS 13 and later
 
-*   For devices that do not support "Force Touch" (such as the iPhone 6 and below, iPhone SE, iPhone XR, iPhone 11 and later, and some iPad models):
+* All devices support notification expanding by performing a right to left swipe and pressing 'View' in the lock screen or pressing and holding, but on 3D Touch-enabled devices you may still need to apply some force to do it. If you're not in the lock screen, you can also pull the notification down to expand it.
```
Comment (TomBrien) [inline docs/notifications/actionable.md]:
```suggestion
* All devices support notification expanding by performing a right to left swipe and pressing 'View' in the lock screen or pressing and holding the notification. If you're not in the lock screen, you can also pull the notification banner down to expand it.
```

**Comment by TomBrien** [comment]
Cheers thanks for contributing to Home Assistant

**Pull request merged**